__tests__/integration/nanocontracts/bet.test.js

```diff
@@ -301,6 +301,10 @@ describe('full cycle of bet nano contract', () => {
     for (const tx of ncHistory2.history) {
       expect(txIds).toContain(tx.hash);
     }
+
+    // Get tx history, it must be greater then zero
+    const txHistory = await wallet.getTxHistory();
+    expect(txHistory).not.toHaveLength(0);
```

Comment:
Why don't you expect the correct number of transactions, which is 5?

Reply:
I can update it, but the purpose was only to call `getTxHistory` with success, and this expect is sufficient for this purpose.